doc/source/conf.py

```diff
@@ -68,7 +68,7 @@
 extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
-    "sphinx.ext.autosectionlabel",
+    # "sphinx.ext.autosectionlabel",
```

Comment:
The lines relative to autoselectionlabel can be deleted

Reply:
Fixed